web/src/app/administration/components/classification-details/classification-details.component.ts

```diff
@@ -137,7 +135,7 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
   onRestore() {
     this.formsValidatorService.formSubmitAttempt = false;
     if (this.action === ACTION.CREATE) {
-      this.classification = new ClassificationDefinition();
+      this.classification = { parentId: this.classification.parentId, parentKey: this.classification.parentKey };
```

Comment:
We should not only keep these two properties, we should also keep the category, type, creation time, domain and  modified time = creation time when restoring. In case you don't want to change here, I could create a new ticket.

Reply:
done